### PR TITLE
docs(blog): independence day

### PR DESCRIPTION
**Pull request by noelweichbrodt**
### PR Checklist

Please review the [guidelines for contributing](CONTRIBUTING.md) to this repository.

- [x] I am requesting to **pull a topic/feature/bugfix branch** (right side). In other words, not _master_.
- [x] I have run `npm test` against my changes and tests pass.
- [x] I have added or edited necessary documentation, or no docs changes are needed.

### Description

New blog post: Independence Day

**Fixes**: #

### CODE REVIEW

File: content/blog/independence-day/independence-day.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+Today, we export to independence!
+
+![Today we celebrate our Independence Day!](https://zippy.gfycat.com/UnkemptInsignificantGelding.webm)
+
+With apologies to President Bill Pullman, let's declare our independence from smart speakers. The voice interface, and the skills you've developed using it, are wonderful resources that are currently stuck in kitchens and dens all over the world. It's time to free them to be carried with your users wherever they are, not wherever Amazon, Google, and Samsung allow them. With Spokestack, your smart speaker skill can declare independence!
+
+## What's independence?
+
+NLU-capable computers everywhere -- your smart phone, your smart watch, your laptop, your tv -- but nobody is building NLU-enabled apps for them. Smart speakers, on the other hand, are essentially NLU-only voice app platforms. Why do you have to choose between them? With Spokestack, you don't. [Export your smart speaker skill](https://spokestack.io/docs/Concepts/export) to run on the major mobile platforms and declare your independence!
```
Comment (space-pope) [inline content/blog/independence-day/independence-day.md]:
```suggestion
NLU-capable computers are everywhere -- your smartphone, your smartwatch, your laptop, your TV — but nobody is building NLU-enabled apps for them. Smart speakers, on the other hand, are essentially NLU-only voice app platforms. Why do you have to choose between them? With Spokestack, you don't. [Export your smart speaker skill](https://spokestack.io/docs/Concepts/export) to run on the major mobile platforms and declare your independence!
```

File: content/blog/independence-day/independence-day.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+date: '2020-06-16'
+description: With Spokestack, your smart speaker skill can declare independence!
+author: noel
+tags: Engineering
+draft: true
+---
+
+Today, we export to independence!
+
+![Today we celebrate our Independence Day!](https://zippy.gfycat.com/UnkemptInsignificantGelding.webm)
```
Comment (space-pope) [inline content/blog/independence-day/independence-day.md]: Don't think the webm is being processed as an image.

File: content/blog/independence-day/independence-day.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+With apologies to President Bill Pullman, let's declare our independence from smart speakers. The voice interface, and the skills you've developed using it, are wonderful resources that are currently stuck in kitchens and dens all over the world. It's time to free them to be carried with your users wherever they are, not wherever Amazon, Google, and Samsung allow them. With Spokestack, your smart speaker skill can declare independence!
+
+## What's independence?
+
+NLU-capable computers everywhere -- your smart phone, your smart watch, your laptop, your tv -- but nobody is building NLU-enabled apps for them. Smart speakers, on the other hand, are essentially NLU-only voice app platforms. Why do you have to choose between them? With Spokestack, you don't. [Export your smart speaker skill](https://spokestack.io/docs/Concepts/export) to run on the major mobile platforms and declare your independence!
+
+![Will Smith walks away from smoke](https://gph.is/28Qnphp)
```
Comment (space-pope) [inline content/blog/independence-day/independence-day.md]: Same here; image link looks broken in preview.

File: content/blog/independence-day/independence-day.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+## Why do I want to declare independence?
+
+Unlike smart speakers, independent voice assistants can take advantage of multiple interfaces modalities to allow your users to get the job done using whatever interaction they find convenient. Visual, haptic, and now voice user interfaces are all accessible to mobile apps using Spokestack.
+
+What do you gain from that? You control and learn from the NLU classification of user utterances, not the smart speaker platform. When you're stuck on a smart speaker, do you get the raw utterance that your user uses when they're in your skill, even misclassified or misunderstood ones? With Spokestack, you control your data and your users data.
```
Comment (space-pope) [inline content/blog/independence-day/independence-day.md]:
```suggestion
What do you gain from that? You control and learn from the NLU classification of user utterances, not the smart speaker platform. When you're stuck on a smart speaker, do you get the raw utterance that your user uses when they're in your skill, even misclassified or misunderstood ones? With Spokestack, you control your data and your users' data.
```
You do get the raw utterance with Dialogflow...just not with Alexa unless you do something clever.

File: content/blog/independence-day/independence-day.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+If you've ever read the EULA for smart speaker platforms, you know how rapacious they are with your data. With Spokestack, you take private the most useful bits of your data instead of feeding it for free to the [FAANG](https://en.wikipedia.org/wiki/Big_Tech).
+
+![Storming the Bastille](https://upload.wikimedia.org/wikipedia/commons/thumb/4/4e/Prise_de_la_Bastille.jpg/800px-Prise_de_la_Bastille.jpg)
+
+## What do I have to do to declare independence?
+
+You can leverage existing ASR and TTS services, which are the parts that actually benefit from scale and are difficult to DIY. Spokestack provides [a seamless, unified API]() across mobile platforms ([iOS](https://spokestack.io/docs/iOS), [Android](https://spokestack.io/docs/Android), and [React/React Native](https://spokestack.io/docs/React%20Native)) that makes integrating your skill into an app easy.
```
Comment (space-pope) [inline content/blog/independence-day/independence-day.md]:
```suggestion
You can leverage existing ASR and TTS services, which are the parts that actually benefit from scale and are difficult to DIY. Spokestack provides a seamless, unified API across mobile platforms ([iOS](https://spokestack.io/docs/iOS), [Android](https://spokestack.io/docs/Android), and [React/React Native](https://spokestack.io/docs/React%20Native)) that makes integrating your skill into an app easy.
```

File: content/blog/independence-day/independence-day.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
+---
+title: Today, we export to independence!
+date: '2020-06-16'
+description: With Spokestack, your smart speaker skill can declare independence!
```
Comment (space-pope) [inline content/blog/independence-day/independence-day.md]:
```suggestion
description: With Spokestack, your voice app can declare independence!
```
I think I'm with Daniel on standardizing around "voice app" (here and elsewhere in the post)—only Alexa apps are "skills", as Google so pedantically reminded me recently.

**Review by space-pope — COMMENTED** [review]
👍

**Comment by timmywil** [comment]
Minor nit: I'd prefer relative URLs where possible.

**Review by timmywil — CHANGES_REQUESTED** [review]
Just need to fix images.

**Review by timmywil — APPROVED** [review]
I went ahead and fixed the images for you. Also, I think I was unclear before. When naming with a folder, the blog post filename should just be `index.md`, and the folder name will match the post title. When not using a folder, the filename would match the post title.

**Comment by timmywil** [comment]
@noelweichbrodt Would you like this out? What date should we set?

**Pull request merged**